manifests/authzd_user.pp

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# keepalived
+#
+# @example
+#   include nebula::profile::authzd_user()
+define nebula::authzd_user(
+  String $home,
+  Hash $key,
+  String $gid) {
+
+  $key_file = "${home}/.ssh/authorized_keys"
+
+  file { $home:
+    ensure => 'directory',
+    mode   => '0755'
+  }
+
+  user { $title:
+    gid     => $gid,
+    home    => $home,
+    require => File[$home]
+  }
+
+  nebula::file::ssh_keys { $key_file:
```

Comment:
Since this is the only place `$key_file` is used, I feel like it'd be fine to skip it and just make this

```
nebula::file::ssh_keys { "${home}/.ssh/authorized_keys":
```

Reply:
I think in a previous iteration it might have been there multiple times. Fixed now.